changed resource file structure, changed code to match

nicola-spire opened the pull request.

nicola-spire commented:
Resource file structure for dashboard matches dropbox now.
Only exception, underscores are replaced with hyphens.

kranar commented:
This pull requests breaks all builds.

On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <notifications@github.com>
wrote:

> Resource file structure for dashboard matches dropbox now.
> Only exception, underscores are replaced with hyphens.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407461999>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGShrExhDd3XRb2d_V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> .
>



-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

nicola-spire commented:
Is it because I changed the folder structure?

On 24 July 2018 at 13:08, kranar <notifications@github.com> wrote:

> This pull requests breaks all builds.
>
> On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <notifications@github.com>
> wrote:
>
> > Resource file structure for dashboard matches dropbox now.
> > Only exception, underscores are replaced with hyphens.
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407461999
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/APzFGShrExhDd3XRb2d_
> V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> > .
> >
>
>
>
> --
> Kamal Mansouri
> Computer Programmer
> +1 647-648-0498
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407480891>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AlFi5sI97KvyHDf2bXsEnqlniEbtaudmks5uJ1SogaJpZM4VdCIp>
> .
>

kranar commented:
I have not looked into why, but the demo's don't work anymore :(

On Tue, Jul 24, 2018 at 1:13 PM, nicola-spire <notifications@github.com>
wrote:

> Is it because I changed the folder structure?
>
> On 24 July 2018 at 13:08, kranar <notifications@github.com> wrote:
>
> > This pull requests breaks all builds.
> >
> > On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <notifications@github.com
> >
> > wrote:
> >
> > > Resource file structure for dashboard matches dropbox now.
> > > Only exception, underscores are replaced with hyphens.
> > >
> > > —
> > > You are receiving this because you are subscribed to this thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/eidolonsystems/nexus/pull/229#
> issuecomment-407461999
> > >,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-
> auth/APzFGShrExhDd3XRb2d_
> > V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> > > .
> > >
> >
> >
> >
> > --
> > Kamal Mansouri
> > Computer Programmer
> > +1 647-648-0498
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407480891
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AlFi5sI97KvyHDf2bXsEnqlniEbtaudmks5uJ1SogaJpZM4VdCIp>
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407482254>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGfPxEQAnKHnKPGoXuAomqbc0SyUvks5uJ1W-gaJpZM4VdCIp>
> .
>



-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

nicola-spire commented:
Changed header-demo to the version I created with the current code. It
works for me!
Only issue: the logo is now a little fuzzy

On 24 July 2018 at 13:21, kranar <notifications@github.com> wrote:

> I have not looked into why, but the demo's don't work anymore :(
>
> On Tue, Jul 24, 2018 at 1:13 PM, nicola-spire <notifications@github.com>
> wrote:
>
> > Is it because I changed the folder structure?
> >
> > On 24 July 2018 at 13:08, kranar <notifications@github.com> wrote:
> >
> > > This pull requests breaks all builds.
> > >
> > > On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <
> notifications@github.com
> > >
> > > wrote:
> > >
> > > > Resource file structure for dashboard matches dropbox now.
> > > > Only exception, underscores are replaced with hyphens.
> > > >
> > > > —
> > > > You are receiving this because you are subscribed to this thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > issuecomment-407461999
> > > >,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-
> > auth/APzFGShrExhDd3XRb2d_
> > > V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> > > > .
> > > >
> > >
> > >
> > >
> > > --
> > > Kamal Mansouri
> > > Computer Programmer
> > > +1 647-648-0498
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/eidolonsystems/nexus/pull/229#
> issuecomment-407480891
> > >,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> > AlFi5sI97KvyHDf2bXsEnqlniEbtaudmks5uJ1SogaJpZM4VdCIp>
> > > .
> > >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407482254
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> APzFGfPxEQAnKHnKPGoXuAomqbc0SyUvks5uJ1W-gaJpZM4VdCIp>
>
> > .
> >
>
>
>
> --
> Kamal Mansouri
> Computer Programmer
> +1 647-648-0498
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407484920>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AlFi5lDkFmjPCgmpzprCO8IeAXVvYFkpks5uJ1ewgaJpZM4VdCIp>
> .
>

kranar commented:
Try this...

Change directory to the root git directory: Nexus

Type the following verbatim:

git clean -fxd

Now rebuild the dashboard demo and run it.

On Tue, Jul 24, 2018 at 1:29 PM, nicola-spire <notifications@github.com>
wrote:

> Changed header-demo to the version I created with the current code. It
> works for me!
> Only issue: the logo is now a little fuzzy
>
> On 24 July 2018 at 13:21, kranar <notifications@github.com> wrote:
>
> > I have not looked into why, but the demo's don't work anymore :(
> >
> > On Tue, Jul 24, 2018 at 1:13 PM, nicola-spire <notifications@github.com>
> > wrote:
> >
> > > Is it because I changed the folder structure?
> > >
> > > On 24 July 2018 at 13:08, kranar <notifications@github.com> wrote:
> > >
> > > > This pull requests breaks all builds.
> > > >
> > > > On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <
> > notifications@github.com
> > > >
> > > > wrote:
> > > >
> > > > > Resource file structure for dashboard matches dropbox now.
> > > > > Only exception, underscores are replaced with hyphens.
> > > > >
> > > > > —
> > > > > You are receiving this because you are subscribed to this thread.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > issuecomment-407461999
> > > > >,
> > > > > or mute the thread
> > > > > <https://github.com/notifications/unsubscribe-
> > > auth/APzFGShrExhDd3XRb2d_
> > > > V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> > > > > .
> > > > >
> > > >
> > > >
> > > >
> > > > --
> > > > Kamal Mansouri
> > > > Computer Programmer
> > > > +1 647-648-0498
> > > >
> > > > —
> > > > You are receiving this because you authored the thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > issuecomment-407480891
> > > >,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-auth/
> > > AlFi5sI97KvyHDf2bXsEnqlniEbtaudmks5uJ1SogaJpZM4VdCIp>
> > > > .
> > > >
> > >
> > > —
> > > You are receiving this because you commented.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/eidolonsystems/nexus/pull/229#
> issuecomment-407482254
> > >,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> > APzFGfPxEQAnKHnKPGoXuAomqbc0SyUvks5uJ1W-gaJpZM4VdCIp>
> >
> > > .
> > >
> >
> >
> >
> > --
> > Kamal Mansouri
> > Computer Programmer
> > +1 647-648-0498
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407484920
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AlFi5lDkFmjPCgmpzprCO8IeAXVvYFkpks5uJ1ewgaJpZM4VdCIp>
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407487502>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGV7LjYCXQOLdFG5TBnLe4QkIW8cEks5uJ1mMgaJpZM4VdCIp>
> .
>



-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

nicola-spire commented:
Ah, I get what's happening. Should work now.

On 24 July 2018 at 13:31, kranar <notifications@github.com> wrote:

> Try this...
>
> Change directory to the root git directory: Nexus
>
> Type the following verbatim:
>
> git clean -fxd
>
> Now rebuild the dashboard demo and run it.
>
> On Tue, Jul 24, 2018 at 1:29 PM, nicola-spire <notifications@github.com>
> wrote:
>
>
> > Changed header-demo to the version I created with the current code. It
> > works for me!
> > Only issue: the logo is now a little fuzzy
> >
> > On 24 July 2018 at 13:21, kranar <notifications@github.com> wrote:
> >
> > > I have not looked into why, but the demo's don't work anymore :(
> > >
> > > On Tue, Jul 24, 2018 at 1:13 PM, nicola-spire <
> notifications@github.com>
> > > wrote:
> > >
> > > > Is it because I changed the folder structure?
> > > >
> > > > On 24 July 2018 at 13:08, kranar <notifications@github.com> wrote:
> > > >
> > > > > This pull requests breaks all builds.
> > > > >
> > > > > On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <
> > > notifications@github.com
> > > > >
> > > > > wrote:
> > > > >
> > > > > > Resource file structure for dashboard matches dropbox now.
> > > > > > Only exception, underscores are replaced with hyphens.
> > > > > >
> > > > > > —
> > > > > > You are receiving this because you are subscribed to this thread.
> > > > > > Reply to this email directly, view it on GitHub
> > > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > > issuecomment-407461999
> > > > > >,
> > > > > > or mute the thread
> > > > > > <https://github.com/notifications/unsubscribe-
> > > > auth/APzFGShrExhDd3XRb2d_
> > > > > V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> > > > > > .
> > > > > >
> > > > >
> > > > >
> > > > >
> > > > > --
> > > > > Kamal Mansouri
> > > > > Computer Programmer
> > > > > +1 647-648-0498
> > > > >
> > > > > —
> > > > > You are receiving this because you authored the thread.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > issuecomment-407480891
> > > > >,
> > > > > or mute the thread
> > > > > <https://github.com/notifications/unsubscribe-auth/
> > > > AlFi5sI97KvyHDf2bXsEnqlniEbtaudmks5uJ1SogaJpZM4VdCIp>
> > > > > .
> > > > >
> > > >
> > > > —
> > > > You are receiving this because you commented.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > issuecomment-407482254
> > > >,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-auth/
> > > APzFGfPxEQAnKHnKPGoXuAomqbc0SyUvks5uJ1W-gaJpZM4VdCIp>
> > >
> > > > .
> > > >
> > >
> > >
> > >
> > > --
> > > Kamal Mansouri
> > > Computer Programmer
> > > +1 647-648-0498
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/eidolonsystems/nexus/pull/229#
> issuecomment-407484920
> > >,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> > AlFi5lDkFmjPCgmpzprCO8IeAXVvYFkpks5uJ1ewgaJpZM4VdCIp>
> > > .
> > >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407487502
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> APzFGV7LjYCXQOLdFG5TBnLe4QkIW8cEks5uJ1mMgaJpZM4VdCIp>
>
> > .
> >
>
>
>
> --
> Kamal Mansouri
> Computer Programmer
> +1 647-648-0498
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407488107>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AlFi5qCkUWO2UcQsa6VJ9dP3jWWw8Z_Jks5uJ1nugaJpZM4VdCIp>
> .
>

kranar commented:
Once you have confirmed that all the tests work as expected, feel free to
submit another commit or just reply to this e-mail if no changes are needed.

On Tue, Jul 24, 2018 at 1:43 PM, nicola-spire <notifications@github.com>
wrote:

> Ah, I get what's happening. Should work now.
>
> On 24 July 2018 at 13:31, kranar <notifications@github.com> wrote:
>
> > Try this...
> >
> > Change directory to the root git directory: Nexus
> >
> > Type the following verbatim:
> >
> > git clean -fxd
> >
> > Now rebuild the dashboard demo and run it.
> >
> > On Tue, Jul 24, 2018 at 1:29 PM, nicola-spire <notifications@github.com>
> > wrote:
> >
> >
> > > Changed header-demo to the version I created with the current code. It
> > > works for me!
> > > Only issue: the logo is now a little fuzzy
> > >
> > > On 24 July 2018 at 13:21, kranar <notifications@github.com> wrote:
> > >
> > > > I have not looked into why, but the demo's don't work anymore :(
> > > >
> > > > On Tue, Jul 24, 2018 at 1:13 PM, nicola-spire <
> > notifications@github.com>
> > > > wrote:
> > > >
> > > > > Is it because I changed the folder structure?
> > > > >
> > > > > On 24 July 2018 at 13:08, kranar <notifications@github.com> wrote:
> > > > >
> > > > > > This pull requests breaks all builds.
> > > > > >
> > > > > > On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <
> > > > notifications@github.com
> > > > > >
> > > > > > wrote:
> > > > > >
> > > > > > > Resource file structure for dashboard matches dropbox now.
> > > > > > > Only exception, underscores are replaced with hyphens.
> > > > > > >
> > > > > > > —
> > > > > > > You are receiving this because you are subscribed to this
> thread.
> > > > > > > Reply to this email directly, view it on GitHub
> > > > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > > > issuecomment-407461999
> > > > > > >,
> > > > > > > or mute the thread
> > > > > > > <https://github.com/notifications/unsubscribe-
> > > > > auth/APzFGShrExhDd3XRb2d_
> > > > > > V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> > > > > > > .
> > > > > > >
> > > > > >
> > > > > >
> > > > > >
> > > > > > --
> > > > > > Kamal Mansouri
> > > > > > Computer Programmer
> > > > > > +1 647-648-0498
> > > > > >
> > > > > > —
> > > > > > You are receiving this because you authored the thread.
> > > > > > Reply to this email directly, view it on GitHub
> > > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > > issuecomment-407480891
> > > > > >,
> > > > > > or mute the thread
> > > > > > <https://github.com/notifications/unsubscribe-auth/
> > > > > AlFi5sI97KvyHDf2bXsEnqlniEbtaudmks5uJ1SogaJpZM4VdCIp>
> > > > > > .
> > > > > >
> > > > >
> > > > > —
> > > > > You are receiving this because you commented.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > issuecomment-407482254
> > > > >,
> > > > > or mute the thread
> > > > > <https://github.com/notifications/unsubscribe-auth/
> > > > APzFGfPxEQAnKHnKPGoXuAomqbc0SyUvks5uJ1W-gaJpZM4VdCIp>
> > > >
> > > > > .
> > > > >
> > > >
> > > >
> > > >
> > > > --
> > > > Kamal Mansouri
> > > > Computer Programmer
> > > > +1 647-648-0498
> > > >
> > > > —
> > > > You are receiving this because you authored the thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > issuecomment-407484920
> > > >,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-auth/
> > > AlFi5lDkFmjPCgmpzprCO8IeAXVvYFkpks5uJ1ewgaJpZM4VdCIp>
> > > > .
> > > >
> > >
> > > —
> > > You are receiving this because you commented.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/eidolonsystems/nexus/pull/229#
> issuecomment-407487502
> > >,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> > APzFGV7LjYCXQOLdFG5TBnLe4QkIW8cEks5uJ1mMgaJpZM4VdCIp>
> >
> > > .
> > >
> >
> >
> >
> > --
> > Kamal Mansouri
> > Computer Programmer
> > +1 647-648-0498
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407488107
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AlFi5qCkUWO2UcQsa6VJ9dP3jWWw8Z_Jks5uJ1nugaJpZM4VdCIp>
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407492038>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGY4l-kan4hud7C7d7vPBnGbCVtZsks5uJ1y1gaJpZM4VdCIp>
> .
>



-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

nicola-spire commented:
Just tested everything. Updated the demos, everything looks fine.

On 24 July 2018 at 13:48, kranar <notifications@github.com> wrote:

> Once you have confirmed that all the tests work as expected, feel free to
> submit another commit or just reply to this e-mail if no changes are
> needed.
>
> On Tue, Jul 24, 2018 at 1:43 PM, nicola-spire <notifications@github.com>
> wrote:
>
>
> > Ah, I get what's happening. Should work now.
> >
> > On 24 July 2018 at 13:31, kranar <notifications@github.com> wrote:
> >
> > > Try this...
> > >
> > > Change directory to the root git directory: Nexus
> > >
> > > Type the following verbatim:
> > >
> > > git clean -fxd
> > >
> > > Now rebuild the dashboard demo and run it.
> > >
> > > On Tue, Jul 24, 2018 at 1:29 PM, nicola-spire <
> notifications@github.com>
> > > wrote:
> > >
> > >
> > > > Changed header-demo to the version I created with the current code.
> It
> > > > works for me!
> > > > Only issue: the logo is now a little fuzzy
> > > >
> > > > On 24 July 2018 at 13:21, kranar <notifications@github.com> wrote:
> > > >
> > > > > I have not looked into why, but the demo's don't work anymore :(
> > > > >
> > > > > On Tue, Jul 24, 2018 at 1:13 PM, nicola-spire <
> > > notifications@github.com>
> > > > > wrote:
> > > > >
> > > > > > Is it because I changed the folder structure?
> > > > > >
> > > > > > On 24 July 2018 at 13:08, kranar <notifications@github.com>
> wrote:
> > > > > >
> > > > > > > This pull requests breaks all builds.
> > > > > > >
> > > > > > > On Tue, Jul 24, 2018 at 12:07 PM, nicola-spire <
> > > > > notifications@github.com
> > > > > > >
> > > > > > > wrote:
> > > > > > >
> > > > > > > > Resource file structure for dashboard matches dropbox now.
> > > > > > > > Only exception, underscores are replaced with hyphens.
> > > > > > > >
> > > > > > > > —
> > > > > > > > You are receiving this because you are subscribed to this
> > thread.
> > > > > > > > Reply to this email directly, view it on GitHub
> > > > > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > > > > issuecomment-407461999
> > > > > > > >,
> > > > > > > > or mute the thread
> > > > > > > > <https://github.com/notifications/unsubscribe-
> > > > > > auth/APzFGShrExhDd3XRb2d_
> > > > > > > V2eFYykKRvz8ks5uJ0ZPgaJpZM4VdCIp>
> > > > > > > > .
> > > > > > > >
> > > > > > >
> > > > > > >
> > > > > > >
> > > > > > > --
> > > > > > > Kamal Mansouri
> > > > > > > Computer Programmer
> > > > > > > +1 647-648-0498
> > > > > > >
> > > > > > > —
> > > > > > > You are receiving this because you authored the thread.
> > > > > > > Reply to this email directly, view it on GitHub
> > > > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > > > issuecomment-407480891
> > > > > > >,
> > > > > > > or mute the thread
> > > > > > > <https://github.com/notifications/unsubscribe-auth/
> > > > > > AlFi5sI97KvyHDf2bXsEnqlniEbtaudmks5uJ1SogaJpZM4VdCIp>
> > > > > > > .
> > > > > > >
> > > > > >
> > > > > > —
> > > > > > You are receiving this because you commented.
> > > > > > Reply to this email directly, view it on GitHub
> > > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > > issuecomment-407482254
> > > > > >,
> > > > > > or mute the thread
> > > > > > <https://github.com/notifications/unsubscribe-auth/
> > > > > APzFGfPxEQAnKHnKPGoXuAomqbc0SyUvks5uJ1W-gaJpZM4VdCIp>
> > > > >
> > > > > > .
> > > > > >
> > > > >
> > > > >
> > > > >
> > > > > --
> > > > > Kamal Mansouri
> > > > > Computer Programmer
> > > > > +1 647-648-0498
> > > > >
> > > > > —
> > > > > You are receiving this because you authored the thread.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > > issuecomment-407484920
> > > > >,
> > > > > or mute the thread
> > > > > <https://github.com/notifications/unsubscribe-auth/
> > > > AlFi5lDkFmjPCgmpzprCO8IeAXVvYFkpks5uJ1ewgaJpZM4VdCIp>
> > > > > .
> > > > >
> > > >
> > > > —
> > > > You are receiving this because you commented.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/eidolonsystems/nexus/pull/229#
> > issuecomment-407487502
> > > >,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-auth/
> > > APzFGV7LjYCXQOLdFG5TBnLe4QkIW8cEks5uJ1mMgaJpZM4VdCIp>
> > >
> > > > .
> > > >
> > >
> > >
> > >
> > > --
> > > Kamal Mansouri
> > > Computer Programmer
> > > +1 647-648-0498
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/eidolonsystems/nexus/pull/229#
> issuecomment-407488107
> > >,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> > AlFi5qCkUWO2UcQsa6VJ9dP3jWWw8Z_Jks5uJ1nugaJpZM4VdCIp>
> > > .
> > >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407492038
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/APzFGY4l-
> kan4hud7C7d7vPBnGbCVtZsks5uJ1y1gaJpZM4VdCIp>
>
> > .
> >
>
>
>
> --
> Kamal Mansouri
> Computer Programmer
> +1 647-648-0498
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/229#issuecomment-407493545>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AlFi5sfj_wnxK5Vf2vzXbSlGYkTqzgjmks5uJ13YgaJpZM4VdCIp>
> .
>
